torch/csrc/jit/codegen/cuda/lower2device.cpp

```diff
@@ -275,6 +275,12 @@ class GpuLower::KernelIrMapper : private OptInConstDispatch {
     TORCH_CHECK(gpu_lower_->kir_expr_map_.insert({node, lowered_node}).second);
   }
 
+  void handle(const TransposeOp* node) final {
```

Comment:
Should we have a lowered version of TransposeOp? Do we have one for broadcast? Might be needed for when we move more of the lowering logic to operate on kir instead of fusion ir.

Reply:
This PR initially had `kir::TransposeOp` as well. After discussing with @tlemo, I realized that's probably not sufficient for plugging in manually-written optimized kernels, so I dropped it. We should revisit it when needed in the future. 

See https://github.com/csarofeen/pytorch/pull/559#discussion_r538742890.